src/interpreter/utils.ts

```diff
@@ -48,6 +44,73 @@ export function composeVersion(version: VersionStructure): string {
   );
 }
 
+function formatPrimitive(value: string | number | boolean): string {
+  if (typeof value === 'string') {
+    return `"${value}"`;
+  }
+
+  return value.toString();
+}
+
+function formatStructure(structure: StructureType | string): string {
+  if (typeof structure === 'string') {
+    return structure;
+  }
+
+  switch (structure.kind) {
+    case 'EnumStructure':
+      return structure.enums.map(enumValue => enumValue.value).join(' or ');
+    case 'ListStructure':
+      return `[${formatStructure(structure.value)}]`;
+    case 'NonNullStructure':
+      return `NonNull ${formatStructure(structure.value)}`;
+    case 'ObjectStructure':
+      return `{ ${Object.entries(structure.fields ?? [])
+        .map(([key, type]) => `${key}: ${formatStructure(type)}`)
+        .join(', ')} }`;
+    case 'PrimitiveStructure':
+      return structure.type;
```

Comment:
maybe formatPrimitive?

Reply:
That is actually string in all cases, its type is `'string' | 'number' | 'boolean'`. Now that you mention it, I always add quotes at `formatIssue` function, so I'll remove `formatPrimitive`, to not cause confusion.

Reply:
Ok, now I understand why I put it there 😄, makes more sense to add quotes to strings when formatting objects. I'll keep `formatPrimitive`, but remove quotes from `formatIssue`.